src/modules/trade/types/TradeDerivedState.ts

```diff
@@ -6,6 +6,18 @@ export interface TradeDerivedState {
   readonly outputCurrency: Currency | null
   readonly inputCurrencyAmount: CurrencyAmount<Currency> | null
   readonly outputCurrencyAmount: CurrencyAmount<Currency> | null
+  /**
+   * Since Cow Protocol supports buying trade type
+   * Sometimes we need a maximum amount to sell
+   *
+   * How it works:
+   * You want to buy exactly 2 WETH for <some amount> of COW.
+   * In this case, you will input 2 WETH in the buy input
+   * and the sell input will be automatically filled by the quote API response, for example with the value 552 COW.
+   * Since the market is liquid, there might be a slippage, let's assume it's 2%.
+   * So, the final sell amount will be 552 COW + 2% = 563.04
+   */
+  readonly sellAmountMaxSpend: CurrencyAmount<Currency> | null
```

Comment:
What about naming it `slippageAdjustedSellAmount`?

You can also add to the docstring info that for Sell orders and for any type of Limit orders, this is always equal to sell amount

Reply:
Thank you! Renamed